project/AppDependencies.scala

```diff
@@ -4,7 +4,7 @@ object AppDependencies {
 
   private val bootstrapPlay26Version = "1.3.0"
   private val xmlResolverVersion = "1.2"
-  private val catsVersion = "2.1.0"
+  private val catsVersion = "2.0.0"
   private val hmrcTestVersion = "3.9.0-play-26"
   private val scalaTestVersion = "3.0.8"
```

Comment:
you could remove scalatest and let scalatestplus pull in that dependency

Reply:
Oddly unfortunately not. 'scalatestplus' lib pulls in 'scalatest' ver 3.0.5, 'customs-api-common' specifically pulls in ver 3.0.8. If I remove the dependency on '3.0.8' we get the error "object mockito is not a member of package org.scalatestplus"